Homework2/Task02/task02.py

```diff
@@ -0,0 +1,23 @@
+import collections
+from typing import List, Tuple
+
+
+def major_and_minor_elem(inp: List) -> Tuple[int, int]:
+    """
+    Given an array of size n, find the most common and the least common elements.
+
+    The most common element is the element that appears more than n // 2 times.
+    The least common element is the element that appears fewer than other.
+    You may assume that the array is non-empty and the most common element
+    always exist in the array.
+    Example 1:
+    Input: [3,2,3]
+    Output: 3, 2
+    Example 2:
+    Input: [2,2,1,1,1,2,2]
+    Output: 2, 1
+    """
+    elements = collections.Counter(inp).most_common()
+    maximum = elements[0][0]
+    minimum = elements[len(elements) - 1][0]
```

Comment:
What is the proper way to access the last element in the collection?

Can you solve it without Counter?

Reply:
Yes, it should be [-1].
I can solve it with dict, but i did something like this in Task01.  So I thought that in this task we can use not only default collections .